packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -102,6 +102,12 @@ describe('internal-plugin-metrics', () => {
       sinon.restore();
     });
 
+    describe('#validator', () => {
```

Comment:
Need a test to validate that `this.validator()` is called by the consuming function.

Reply:
good call, will add

Reply:
done